modules/claims_api/spec/lib/claims_api/service_branch_mapper_spec.rb

```diff
@@ -25,6 +25,7 @@
   'Marine Corps Reserves' => 'Marine Corps Reserves',
   'Merchant Marine' => 'Merchant Marine',
   'National Oceanic and Atmospheric Administration' => 'National Oceanic & Atmospheric Administration',
+  'National Oceanic &amp; Atmospheric Administration' => 'National Oceanic & Atmospheric Administrations',
   'Naval Academy' => 'Naval Academy',
```

Comment:
oops, `Administration` is plural here

Reply:
Sorry, was testing running rspec and thought I changed that back.